universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java

```diff
@@ -119,6 +119,7 @@ public ActiveAndDraftPrograms getActiveAndDraftPrograms() {
     validateProgramText(errorsBuilder, "admin description", adminDescription);
     validateProgramText(errorsBuilder, "display name", defaultDisplayName);
     validateProgramText(errorsBuilder, "display description", defaultDisplayDescription);
+    validateProgramText(errorsBuilder, "display mode", displayMode);
```

Comment:
suggestion (blocking): please write a unit test that covers this change in behavior. see https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/test/services/program/ProgramServiceImplTest.java